cluster/manifests/02-admission-control/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       priorityClassName: system-cluster-critical
       containers:
       - name: admission-controller
-        image: 926694233939.dkr.ecr.eu-central-1.amazonaws.com/production_namespace/teapot/admission-controller:master-206
```

Comment:
This is the only related change, the rest should go away once we merge `dev-to-eks` branch.

Reply:
Rebased on latest `eks` after the merge, and now it's as expected.